docs/patternlib/tutorials/workshop.md

```diff
@@ -593,33 +593,37 @@ You can also control external synthesizers by [MIDI](/configuration/MIDIOSC/MIDI
 
 ### Difference between functions `n` and `note`
 
-When using synths, both functions are exactly the same: you may have noticed that the above example plays a C note, an E note (which is 4 semitones above C), and a G note (which is 7 semitones above C). This is exactly the same as `d1 $ note "0 4 7" # sound "superpiano"`.
+When using synths, both `n` and `note` functions are exactly the same: you may have noticed that the above example plays a C note, an E note (which is 4 semitones above C), and a G note (which is 7 semitones above C). This is exactly the same as:
 
-When using samples, `n` refers to sample number (which eventually can be a note, if you have sampled every single note of an instrument). However, `note` pitches the sample up or down (and the sample duration is affected accordingly).
+```haskell
+d1 $ note "0 4 7" # sound "superpiano"
+```
+
+When using samples, `n` refers to the file index in the sample folder, sorted alphabetically (ascending) and counted from 0 (zero). It is possible for each sample to correspond to a note, if you have sampled every single note of an instrument. However when using `note`, the sample is pitched up or down (and the sample duration is affected accordingly).
 
 So, for example:
 ```haskell
 d1 $ sound "bd*4" # n "<0 4>" # note "0 12 -7 -12"
 ```
 
-This will play the first `bd` sample on odd cycles and the fifth on even cycles. On each cycle, the sample will be played 4 times: one as is, one pitched an octave above (12 semitones), one a fifth below (7 semitones), and the last one an octave below.
+This will play the first sample in the `bd` folder on odd cycles and the fifth sample on even cycles. On each cycle, the sample will be played 4 times: one as is, one pitched an octave above (12 semitones), one a fifth below (7 semitones), and the last one an octave below.
 
 -----
 
 ## Playing notes
 
 Most of this tutorial is dedicated to rhythm, but **Tidal Cycles** also offers ways to play notes, scales, chords and arpeggios.
 
-You already know how to play notes: using the `note` function or, in case you have a sampled instrument, choosing notes with the `n` function.
+You already know how to play notes: using the `note` function or, in case you have a per-note sampled instrument, choosing notes with the `n` function.
 
-You can also write directly note names. For example, these two codes are equivalent:
+You can also write notes based on the Western Music Theory naming convention which uses the first 7 letters of the alphabet (A to G). For example, these two codes are equivalent:
```

Comment:
I'm not sure about this. The naming A..G is not the only one used in western music theory. In many places we use Do, Re .. Si naming.

Reply:
I believe the do-re etc ("solfege") is referred to as "European" Music theory - I'm relying heavily on wikipedia for this though

Reply:
Ok, in either case, it's clearer with this explanation